api/src/paths/project/{projectId}/survey/{surveyId}/observation/submission/get.ts

```diff
@@ -87,33 +102,62 @@ export function getObservationSubmission(): RequestHandler {
     const connection = getDBConnection(req['keycloak_token']);
 
     try {
-      const getObservationSubmissionSQLStatement = getLatestSurveyOccurrenceSubmissionSQL(Number(req.params.surveyId));
+      const getOccurrenceSubmissionSQLStatement = getLatestSurveyOccurrenceSubmissionSQL(Number(req.params.surveyId));
 
-      if (!getObservationSubmissionSQLStatement) {
-        throw new HTTP400('Failed to build SQL get statement');
+      if (!getOccurrenceSubmissionSQLStatement) {
+        throw new HTTP400('Failed to build SQL getLatestSurveyOccurrenceSubmissionSQL statement');
       }
 
       await connection.open();
 
-      const observationSubmissionData = await connection.query(
-        getObservationSubmissionSQLStatement.text,
-        getObservationSubmissionSQLStatement.values
+      const occurrenceSubmissionData = await connection.query(
+        getOccurrenceSubmissionSQLStatement.text,
+        getOccurrenceSubmissionSQLStatement.values
       );
 
+      let messageList = [];
+
+      if (
+        occurrenceSubmissionData &&
+        occurrenceSubmissionData.rows &&
+        occurrenceSubmissionData.rows[0] &&
+        occurrenceSubmissionData.rows[0].submission_status_type_name === 'Rejected'
```

Comment:
Right now we only insert error messages, but eventually we'll have non-error messages as well, so we'll need to update this to return all messages, and update the frontend to parse between the different status types.  But, that can be a future change.